pattern-categorization/innersource-program-mind-map.md

```diff
@@ -2,90 +2,106 @@
 
 ## Begin
 
-### [Start as an Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md)
+### Program Setup
 
-### [Dedicated Community Manager](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/dedicated-community-leader.md)
+#### Management hesitates to invest in InnerSource
+
+##### [Start as an Experiment](https://patterns.innersourcecommons.org/p/start-as-experiment)
+
+#### Slow community growth hinders InnerSource
+
+##### [Dedicated Community Leader](https://patterns.innersourcecommons.org/p/dedicated-community-leader)
 
 ### Project Setup
 
-#### [Standard Base Documentation](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/project-setup/base-documentation.md)
+#### Quickly assessing projects of different teams is hard
+
+##### [Standard Base Documentation](https://patterns.innersourcecommons.org/p/base-documentation)
 
-#### [Communication Tooling](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/project-setup/communication-tooling.md)
+#### Difficulties in requesting support and contacting the project team
```

Comment:
How about `ad-hoc communication hinders project growth`? That might match closer to the pattern's problem description?

Reply:
Good one. Done!